pkg/providers/openstack/image.go

```diff
@@ -32,10 +32,10 @@ import (
 	"github.com/gophercloud/gophercloud/v2"
 	"github.com/gophercloud/gophercloud/v2/openstack"
 	"github.com/gophercloud/gophercloud/v2/openstack/image/v2/images"
+	"github.com/kaptinlin/jsonschema"
```

Comment:
Technically there is a kube-openapi package, to keep things in the "more trustworthy" ecosystem, but this library is way simpler and not over-engineered like everything Kubernetes!